x-pack/plugins/security_solution/public/resolver/view/index.tsx

```diff
@@ -7,50 +7,29 @@
 
 import React, { useMemo } from 'react';
 import { Provider } from 'react-redux';
-import { ResolverMap } from './map';
 import { storeFactory } from '../store';
 import { StartServices } from '../../types';
 import { useKibana } from '../../../../../../src/plugins/kibana_react/public';
+import { DataAccessLayer, ResolverProps } from '../types';
+import { dataAccessLayerFactory } from '../data_access_layer/factory';
+import { ResolverWithoutProviders } from './resolver_without_providers';
 
 /**
- * The top level, unconnected, Resolver component.
+ * The `Resolver` component to use. This sets up the DataAccessLayer provider. Use `ResolverWithoutStore` in tests or in other scenarios where you want to provide a different (or fake) data access layer.
```

Comment:
:grey_question: You mean `ResolverWithoutProviders` right?

Reply:
fixed